src/rgw/rgw_admin.cc

```diff
@@ -5504,6 +5508,86 @@ int main(int argc, const char **argv)
     }
   }
 
+  if (opt_cmd == OPT_BUCKET_CHOWN) {
+    int ret = 0;
+    std::string tenant;
+    std::vector<rgw_bucket_dir_entry> objs;
+    map<string, bool> common_prefixes;
+    RGWObjectCtx obj_ctx(store);
+
+    //Get bucket info
+    RGWBucketInfo bucket_info;
+    RGWSysObjectCtx sys_ctx = store->svc.sysobj->init_obj_ctx();
+    map<string, bufferlist> attrs;
+
+    //split tenant/bucket
```

Comment:
is this done elsewhere?  would be nice to have a common routine for this name/tenant splitting